tests/testTriggers.go

```diff
@@ -647,6 +649,70 @@ func TriggerRestartVolDriver(contexts *[]*scheduler.Context, recordChan *chan *E
 	})
 }
 
+// TriggerRestartKvdbVolDriver restarts volume driver where kvdb resides and validates app
+func TriggerRestartKvdbVolDriver(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: RestartKvdbVolDriver,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	Step("get kvdb nodes bounce volume driver", func() {
+		for _, appNode := range node.GetMetadataNodes() {
+			Step(
+				fmt.Sprintf("stop volume driver %s on node: %s",
+					Inst().V.String(), appNode.Name),
+				func() {
+					taskStep := fmt.Sprintf("stop volume driver on node: %s.",
+						appNode.MgmtIp)
+					event.Event.Type += "<br>" + taskStep
+					errorChan := make(chan error, errorChannelSize)
+					StopVolDriverAndWait([]node.Node{appNode}, &errorChan)
+					for err := range errorChan {
+						UpdateOutcome(event, err)
+					}
+				})
+
```

Comment:
can we have a wait and validation here that  some other helathy node is picked up kvdb node or 
restart master kvdb node and other node is made as master